packages/sdk/src/index.ts

```diff
@@ -68,10 +68,10 @@ async function scheduleXcmpTaskWithPayThroughRemoteDerivativeAccountFlow({
   keyringPair,
 }: ScheduleXcmpTaskWithPayThroughRemoteDerivativeAccountFlowParams): Promise<SendExtrinsicResult> {
   const oakApi = oakAdapter.getApi();
-  const { paraId, xcmInstructionNetworkType, relayChain } = destinationChainAdapter.getChainData();
+  const { paraId, xcmInstructionNetworkType, network } = destinationChainAdapter.getChainData();
   if (!paraId) throw new Error("paraId not set");
   if (!xcmInstructionNetworkType) throw new Error("xcmInstructionNetworkType not set");
-  if (!relayChain) throw new Error("relayChain not set");
+  if (!network) throw new Error("relayChain not set");
```

Comment:
why this line is checking `network` but the error message is **relayChain** not set?